docassemble/assemblylinewizard/generator_constants.py

```diff
@@ -132,6 +132,7 @@ class Object(object):
   '_address_line_one': ".address.line_one()",
   '_address_city_state_zip': ".address.line_two()",
   '_signature': ".signature",
+  '_mail_address': ".mail_address"
```

Comment:
We also need to handle all of the variations on the mail_address object.

E.g.: mail_address.block(), mail_address.on_one_line(), mail_address.line_one(), mail_address.line_two(), mail_address.city, .zip, .state, etc.